### PR TITLE
added Multus logging and few minor changes

### PR DESCRIPTION
**Pull request by naiming-zededa**
- add logging and rotation for multus

### CODE REVIEW

File: pkg/kube/cluster-init.sh [hunk 1]
```diff
@@ -29,16 +28,16 @@ setup_cgroup () {
...
     fi
-    if [ -f "$k3slogFile.1" ]; then
-      cp "$k3slogFile.1" "$k3slogFile.2"
+    if [ -f "$K3S_LOG_DIR/$1.1" ]; then
+      cp "$K3S_LOG_DIR/$1.1" "$K3S_LOG_DIR/$1.2"
     fi
-    cp "$k3slogFile" "$k3slogFile.1"
-    cat /dev/null > "$k3slogFile"
+    cp "$K3S_LOG_DIR/$1" "$K3S_LOG_DIR/$1.1"
+    truncate -s 0 "$K3S_LOG_DIR/$1"
```
Comment (zedi-pramodh) [inline pkg/kube/cluster-init.sh:40]: Why do we need that ? If log rotate is in place ?

Comment (zedi-pramodh) [reply pkg/kube/cluster-init.sh:40]: Oh its not real k3s based logrotate ?  Looks like poor mans logrotate :)

**Review by zedi-pramodh — APPROVED** [review]
LGTM

File: pkg/kube/cluster-init.sh (hunk 1 above)
Comment (naiming-zededa) [reply pkg/kube/cluster-init.sh:40]: Yeh, since i have already implemented that, and now it handles other logs too, such as multus.log

**Pull request merged**